src/data/content.json

```diff
@@ -0,0 +1,27 @@
+[
+  {
+    "id": "01",
+    "headline": "2 MINUTEN…",
+    "title": "Atemzüge zählen",
+    "image": "/images/atemzuege.jpg",
+    "description1": "Wähle einen aufrechten bequemen Sitz. Schließ deine Augen oder fokussiere deinen Blick auf einen Punkt der sich nicht bewegt. Ateme tief in deinem gesamten Brustkorb ein und wieder aus. Wiederhole dies drei mal ohne zu zählen. ",
+    "description2": "Bei dem nächsten Atemzug zähle bei der Einatmung und Ausatmung jeweils bis vier. Jeder Atemzug hat die länge von eins - vier. Einatmen in den gesamten Brustkorb zähle bis vier, Ausatmen Brustkorb senkt sich zähle bis vier.",
+    "description3": "Stell dir einen Timer auf zwei Minuten. Beobachte nach der Übung noch für einen kurzen Moment, wie sich dein natürlicher Atem jetzt anfühlt."
+  },
+  {
+    "id": "03",
+    "headline": "10 MINUTEN…",
+    "title": "Bodyscan",
+    "description1": "Setz dich auf einen Stuhl. Die Beine sind angewinkelt, die Füsse berühren den Boden. Lenk deinen Fokus auf dein inneres deine Gedanken und dein Körper. Wie fühlt sich dein Körper jetzt an, welche Gedanken kommen. Nimm diese nur wahr ohne sie zu bewerten.",
+    "description2": "Beginn damit gedanklich durch deinen Körper zu wandern. Starte bei den Füssen. Achte bei jeder Körperregion genau darauf sie sie sich anfühlt. Sei bewusst dabei. Wenn Gedanken aufkommen nimm diese war, schweif nicht ab in die Gedanken.",
+    "description3": "Die letzten Minuten des Bodyscan, sei entspannt. Du musst dich auf nichts mehr konzentrieren, lass alle Gedanken und Wahrnehmungen los. Zum Abschluss lenk deinen Fokus noch einmal auf deinen Körper, nimm wahr wie er sich jetzt anfühlt."
+  },
```

Comment:
In this object the image is missing.

Reply:
it is so right. :) 
Some one with and some one without image